modules/@apostrophecms/home-page/views/page.html

```diff
@@ -9,9 +9,13 @@
 {% block sideNav %}{% endblock %}
 
 {% block main %}
-  <section class="latest-article">
-    <h2>Latest Article</h2>
-    {% component "article:recent" with { limit: 1 } %}
-  </section>
-  {% area data.page, 'main' %}
+  <div class="home">
+    <section class="latest-article">
```

Comment:
Looking at the `recent` snippet and this it seems like it'd be better n to put this whole `section` in the template component so it can always be assured to have consistent structure and semantics.